mods/creative/inventory.lua

```diff
@@ -172,9 +180,5 @@ creative.register_tab("craftitems", "Items", minetest.registered_craftitems)
 
 local old_homepage_name = sfinv.get_homepage_name
 function sfinv.get_homepage_name(player)
-	if minetest.setting_getbool("creative_mode") then
-		return "creative:all"
-	else
-		return old_homepage_name(player)
-	end
+	return creative_mode and "creative:all" or old_homepage_name(player)
```

Comment:
@sapier it should provide the same result if creative_mode isn't changed during play. However, as it is less readable, I'd prefer this syntax would not be used for ternary conditionals (especially that it can be tricky, i.e. `a and b or c` is *not* the same as `a?b:c` if `b` evaluates to a false-y value).